packages/api-gateway-service/src/helpers.ts

```diff
@@ -2,10 +2,10 @@ const fetch = require( 'node-fetch' );
 
 /* This consists of the list of microservices which needs to be part of the gateway */
 export const serviceList = [
-  { name: 'User Service', url: `http://${ process.env.USER_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Home Service', url: `http://${ process.env.HOME_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Feedback Service', url: `http://${ process.env.FEEDBACK_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Notification Service', url: `http://${ process.env.NOTIFICATION_SERVICE_SERVICE_HOST }/graphql` },
+  { name: 'User Service', url: `http://${ process.env.USER_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Home Service', url: `http://${ process.env.HOME_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Feedback Service', url: `http://${ process.env.FEEDBACK_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Notification Service', url: `http://${ process.env.NOTIFICATION_SERVICE_SERVICE_HOST }:8080/graphql` },
```

Comment:
A small change to the variable name for Notifications. Since the package name in the project is `notifications-service`

```suggestion
  { name: 'Notification Service', url: `http://${ process.env.NOTIFICATIONS_SERVICE_SERVICE_HOST }:8080/graphql` },
```